### ISSUE TITLE
[Icon Container] There is no possibility click on icon container from keyboard

### ISSUE THREAD

**Issue by NatalliaAlieva** [epam/UUI#906]
## Description
User cannot click on icon container from keyboard.

## Steps to Reproduce
1. Open [Icon Container](https://uui.epam.com/documents?category=components&id=iconContainer&mode=propsEditor&skin=UUI3_loveship) component.
2. Select any icon in the `icon` property.
3. Click on Icon Container.

## Actual result
Loveship: focus doesn't go to the icon, when User using the keyboard.
Promo: focus goes to the icon, when User using the keyboard, but `Enter` key doesn't work on it.

https://user-images.githubusercontent.com/57751283/209693501-8914755e-7380-425c-a7a7-64315bc84cbf.mp4

## Expected result
User can click on the Icon Container using `Enter` key.

## Environment
* UUI version: 4.9.2
* Browser name: any
* Operating System: any

**Comment by NatalliaAlieva** [comment]
The **Icon Container** component should not be used as a clickable component, we have an **Icon Button** component which has a hover/active style for this. onClick in Icon Container is a technical property when you want the icon to actually be clicked - but this implies clicking on some common container - e.g. chevron in pickers.
